walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -44,9 +44,17 @@
         for (Event event: events) {
             if (potentialMeetingTime.overlaps(event.getWhen())) {
                 for (String attendee: request.getAttendees()) {
-                    if (event.getAttendees().contains(attendee)) {
-                        available = false;
-                        break;
+                    if (request.getOptionalAttendees().size() == request.getAttendees().size()) {
```

Comment:
The algorithm has 2 behaviors 
if all attendees are optional then you treat them like they're mandatory attendees by displaying the times that work for optional attendees.
Usually, the optional attendees are ignored when considering available time slots.

So, I check if the optional attendees and mandatory attendees have the same size because if they're the same then that lets me know all attendees are optional.

Reply:
Okay sure.